### PR TITLE
Feat/pupil 844/sign post take down

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

Music year: {owa_music_year}

- List of changes
finally! display banner on lesson listing and lesson pages for units with lessons which will be expired in august

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2632--oak-web-application.netlify.thenational.academy/pupils/programmes/english-primary-year-3-l/units/the-day-the-crayons-quit-persuasive-writing-0c4a/lessons
2. you should see banner with lesson removal warnings
3. Click on "To share and discuss a story"
4. You should see banner saying the lesson will be removed and link to support page

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/PupilViews/PupilLessonListing/PupilLessonListing.view.tsx [hunk 1]
```diff
@@ -35,11 +38,15 @@ export const PupilViewsLessonListing = (props: PupilLessonListingViewProps) => {
...
     phaseSlug,
   } = programmeFields;
+  const [showExpiredLessonsBanner, setShowExpiredLessonsBanner] =
+    useState<boolean>(unitData.expirationDate !== null);
 
   const noneExpiredLessons = orderedCurriculumData.filter(
     (lesson) => !lesson.lessonData?.deprecatedFields?.expired,
   );
 
+  console.log(noneExpiredLessons, "noneExpiredLessons");
```
Comment (kimon-satan) [inline src/components/PupilViews/PupilLessonListing/PupilLessonListing.view.tsx:48]: Cleanup pls

**Review by kimon-satan — APPROVED** [review]
One little thing but no blockers

**Comment by RBrownsett** [comment]
@benprotheroe just a little thing, bit I noticed on the lesson page when the banner is displayed it has introduced a segment of white space at the bottom of the page - when the banner is dismissed it does disappear. Screenshot:

image

**Comment by RBrownsett** [comment]
@benprotheroe ah also the failing percy test of lesson page is because the white space introduced between the subject image and the vie all lessons button 

image

**Pull request merged**